docs/serving/queue-extensions.md

```diff
@@ -0,0 +1,40 @@
+# Extending Queue Proxy image with QPoptions
+
+Knative service pods include two containers:
+
+- The user main service container, which is named `user-container`
+- The Queue Proxy - a sidecar named `queue-proxy` that serves as a reverse proxy in front of the `user-container`
+
+Queue Proxy can be extended to offer additional optional features. The QPOptions feature of Queue Proxy allows additional runtime packages to extend Queue Proxy capabilities.
```

Comment:
```suggestion
You can extend Queue Proxy to offer additional features. The QPOptions feature of Queue Proxy allows additional runtime packages to extend Queue Proxy capabilities.
```

Reply:
Should it be "QPOptions" or "QPoptions" here? Make sure one or the other is consistently used.